Framework/CurveFitting/test/Functions/CrystalFieldControlTest.h

```diff
@@ -347,4 +347,30 @@ class CrystalFieldControlTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(isMultiSpectrum, true);
     TS_ASSERT_EQUALS(nControls, 2);
   }
+
+  void test_build_17() {
```

Comment:
Could you name your test more descriptively?

Reply:
Yes, I could. Probably makes sense to rename the other 16 tests as well...

Reply:
Only if you understand what they're testing 😄

Reply:
The renaming is quite straightforward. Unfortunately void `xtest_build_7()` started to fall over after just renaming. It is not a surprise as there was one unnecessary line in the test code so easy to fix. It just makes me wonder how this test was not failing before as it should have failed.

Reply:
I think the `x` at the front of the old name `xtest_build_7` means that the test wasn't being run previously